loggroup-lambda-connector/Readme.md

```diff
@@ -1,29 +1,8 @@
 # LogGroup Lambda Connector
 This is used to automatically subscribe newly created Cloudwatch LogGroups to a Lambda function.
 
-### Prerequisites
```

Comment:
We can keep these but under a new section, e.g: For developers. Essentially everything needed to build for them.